### ISSUE TITLE
Alter user account filters and actions by role

### ISSUE THREAD

**Issue by genjohnson** [uiowa/rusty#152]
The "People" page (/admin/people) exposes Actions and Filters which users may not be able to use depending on their role. It'd be nice to clean this up so that the options exposed to each role are only the options they may use.

- Users with the "editor" role may not perform any Action on user accounts, so I would expect that they not be able to see the list of Actions and "Apply to selected items" buttons.
- Users with the "webmaster" role may not perform any Action on "administrator" user accounts, so they shouldn't see the Actions to block/unblock accounts with that role.
- Only users with the "administrator" role may see & manage roles with the "administrator" role, so "editors" and "webmasters" should not be able to filter user accounts by the "administrator" role.
- Exposing the list of Permissions seems like it would open a can of worms (or at least support requests asking why certain roles do not have certain permissions), so I recommend we remove that, too.

**Comment by richardbporter** [comment]
👍 Maybe some of this should be addressed in the module itself?

Also, are you seeing the list of permissions still? I ran fresh install on master but don't see those as editor or webmaster.

**Comment by genjohnson** [comment]
@richardbporter Yes, you're right, most of this (1-3) should probably be addressed in administerusersbyrole. 

We chatted about this in person, but for the record...the list of Permissions I'm referring to is an exposed filter on the page admin/people. I am going to remove this filter in #155

**Comment by genjohnson** [comment]
For milestone 1.0.0, it is enough to:

- Remove  "administrator" from the role drop-down for non-admins.
- Remove the user actions list and buttons for the Editor role.

These changes may be addressed in a more flexible way during a future milestone.
